tests/models/fuyu/test_processing_fuyu.py

```diff
@@ -44,11 +42,80 @@ def test_fuyu_processing(self):
+
+        EXPECTED_IMAGE_PATCH_INPUTS = torch.cat([SINGLE_IMAGE_PATCH_INPUTS, SINGLE_IMAGE_PATCH_INPUTS], dim=0)
+        EXPECTED_PADDED_UNPACKED_TOKEN_INPUTS = torch.cat([SINGLE_PADDED_UNPACKED_TOKEN_INPUTS, SINGLE_PADDED_UNPACKED_TOKEN_INPUTS], dim=0)
+        # fmt: on
+
+        # Batch of two images - equally sized
+        images = [self.bus_image_pil, self.bus_image_pil]
+        processor_outputs = self.processor(text=[self.text_prompt, self.text_prompt], images=images)
+
+        self.assertTrue((processor_outputs["image_patches_indices"] == EXPECTED_IMAGE_PATCH_INPUTS).all())
+        self.assertTrue((processor_outputs["input_ids"] == EXPECTED_PADDED_UNPACKED_TOKEN_INPUTS).all())
+
+        # Batch of two images - different sizes
+        images = [self.bus_image_pil, self.bus_image_pil.resize((64, 300))]
+        processor_outputs = self.processor(text=[self.text_prompt, self.text_prompt], images=images)
+        # FIXME - test outputs
```

Comment:
To be completed, this succeeds now.

Reply:
I've added a test which checks the processing of an individual resized images, and then checks the padding for two differently sized images in a batch